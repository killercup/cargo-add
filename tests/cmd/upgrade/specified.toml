bin.name = "cargo-upgrade"
args = ["upgrade", "a"]
status = "success"
stdout = """
cargo-list-test-fixture:
"""
stderr = """
   Upgrading a v1.0 -> v99999.0
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
