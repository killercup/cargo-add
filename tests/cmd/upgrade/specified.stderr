    Upgrading a v1.0 -> v99999.0.0
